docs/Security.md

```diff
@@ -34,6 +35,47 @@ end
 
 > Please note that this Initializable pattern should only be used on one function.
 
+## Pausable
+
+The Pausable library allows contracts to implement an emergency stop mechanism. This can be useful for scenarios such as preventing trades until the end of an evaluation period and having an emergency switch to freeze all transactions in the event of a large bug.
```

Comment:
```suggestion
The Pausable library allows contracts to implement an emergency stop mechanism. This can be useful for scenarios such as preventing trades until the end of an evaluation period or having an emergency switch to freeze all transactions in the event of a large bug.
```